client/docs/overview/no-sql.md

```diff
@@ -0,0 +1,38 @@
+# Using NoSQL
+
+Ulixee's NoSQL capabilities are still very minimal. You can do straightforward fetches, runs, and crawls, but no complicated structures such as joins are supported at this time. We recommend using Ulixee SQL for more complicated query needs.
```

Comment:
This does not match my understanding of what "noSQL" means. I know we've had this conversation a few times, so I'm not sure there's any action/todo, but NoSQL implies a language or a storage structure to me (a DSL). Simply calling functions and methods is a BIG stretch to fall into that DSL category. I would have just called this Basic API.

Reply:
Agreed. I guess I figured we'll quickly add some basic join structures soon.